metadata.json

```diff
@@ -11,7 +11,7 @@
   "operatingsystem_support": [
     {
     "operatingsystem":"Windows",
-    "operatingsystemrelease":[ "2008R2", "2012", "2012R2" ]
+    "operatingsystemrelease":[ "2008R2", "2012", "2012R2", "7", "8" ]
```

Comment:
Thanks for testing, all I wonder is

 * should this metadata change migrate into its own changeset?
 * do we have special requirements for rigorous testing before making this claim?

As for the latter, I suppose not. If you've seen it working on those platforms - great. If there are issues that were missed, those will come up eventually.